### DISCUSSION TITLE
Extend report to validate values for Parameter Committee Proposal

### DISCUSSION THREAD

**Discussion by fabianbormann** [cardano-foundation/cf-java-rewards-calculation#14] [General]
The current report is not up-to-date with the current epoch, so it should be updated also to be available as a document to validate the values from this Parameter Committee Proposal.

https://forum.cardano.org/t/pcp-treasurytax-tau-parameter-earncoinpool/123245?u=earncoinpool

**Comment by fabianbormann** [comment]
The [PCP_TreasuryTax-tau-parameter_EarnCoinPool](https://forum.cardano.org/t/pcp-treasurytax-tau-parameter-earncoinpool/123245) proposal addresses the treasury growth rate, which adds currently 20% of the [Total Reward Pot at Epoch n](https://github.com/cardano-foundation/cf-java-rewards-calculation/blob/a794130dc0e320426725a58b8b15f1fbe726b2de/src/main/java/org/cardanofoundation/rewards/calculation/TreasuryCalculation.java#L42) to the treasury each epoch. The proposal is to lower this rate, perhaps even to 5%, which was one of the original ideas before Shelley.

It's important to note that there is more than a 20% addition to the treasury due to other rules, such as PoolReap. Additionally, since epoch 400, there has been a significant inflow that I cannot explain yet, as it has nothing to do with MIR certs. ![image](https://github.com/cardano-foundation/cf-java-rewards-calculation/assets/1525818/f32bd1ca-3313-4ba8-a3c5-ac58fad3eead)  

(see: [Treasury Calculation Report](https://cardano-foundation.github.io/cf-java-rewards-calculation/report-latest/treasury_calculation.html))

In the Conway era, there will also be the possibility to add donations to the treasury, which will be another factor increasing the amount going into the treasury for each epoch.

**Comment by fabianbormann** [comment]
I will regenerate the report to have it up-2-date for the current epoch. But also the old one shows that there is more than the actual 20% going into the treasury pot each epoch.

**Reply by EarnCoinPool** [reply]
Hello, sorry I finally have time to get back to this. How do we confirm these numbers?

"Between Epochs 433 and 443 the average reward pot was 26.4M [ rho * reserve + fees ] , the average treasury contribution per epoch was 5.2M [Pots | Cardano Explorer (cexplorer.io) 1], the average distributed to SPO’s and delegators was 10.1M, and the amount returned to the reserve was 11.1M."

I want to confirm the total reward pot per epoch and the total returned to the reserve so we can confirm this statement:

"11.1M was returned to the reserve due to the fraction of unclaimed rewards returned to the reserves.

The effective treasury tax rate is closer to 33%. So approximately 30 Million ADA is added to the Treasury each month."

It seems we have confirmed the rewards to SPOs and delegators at around 10.1 million, now I just want to confirm the rest. Thank you for your help!

The above is from post here:
https://forum.cardano.org/t/pcp-treasurytax-tau-parameter-earncoinpool/123245/3?u=earncoinpool

**Reply by fabianbormann** [reply]
@EarnCoinPool unfortunately I had some other task todo for today but I will provide a detailed answer tomorrow or on Monday thx 🙏 😊

**Reply by EarnCoinPool** [reply]
No worries. Thanks for your help!

**Reply by EarnCoinPool** [reply]
Hello just wanted to follow-up. No rush but wanted to check-in. Thanks
